ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml

```diff
@@ -32392,18 +32392,7 @@ periodics:
     ci-operator.openshift.io/variant: amd64-stable
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-stable-aws-rosa-hcp-byo-kms-oidc-auditlog-stage-critical-f14
-  reporter_config:
-    slack:
-      channel: '#managed-hypershift-ci-watcher'
```

Comment:
Slack notification got removed here. Is it expected ?

Reply:
No, but it is inconvenient to handle this manually.  Just restore them back.